src/components/Comments/ContainerComment.js

```diff
@@ -179,7 +179,9 @@ export const ReplyComment = ({
   );
 };
 export const ContainerReply = ({children, isGrandchild}) => (
-  <View style={[{borderColor: isGrandchild ? COLORS.white : COLORS.balance_gray}]}>{children}</View>
+  <View style={[{borderColor: isGrandchild ? COLORS.almostBlack : COLORS.balance_gray}]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the border color of the `ContainerReply` component when `isGrandchild` is true might affect readability depending on the background color. If the background color is dark, using `COLORS.almostBlack` as a border color might make the border less visible. Please ensure that this color change doesn't negatively impact the UI's readability and accessibility.
```diff
- <View style={[{borderColor: isGrandchild ? COLORS.almostBlack : COLORS.balance_gray}]}>
+ <View style={[{borderColor: isGrandchild ? appropriateColor : COLORS.balance_gray}]}>
```
In the above snippet, replace `appropriateColor` with the color that ensures good contrast with the background color.



<!-- This is an auto-generated comment by OSS CodeRabbit -->